cms/djangoapps/contentstore/views/tests/test_certificates.py

```diff
@@ -295,8 +295,8 @@ def test_certificate_info_in_response(self):
 
         # in html response
         result = self.client.get_html(self._url())
-        self.assertIn('Test certificate', result.content)
-        self.assertIn('Test description', result.content)
+        self.assertIn('Test certificate', result.content.decode('utf-8'))
+        self.assertIn('Test description', result.content.decode('utf-8'))
```

Comment:
Nit: See [FAQ #5](https://openedx.atlassian.net/wiki/spaces/AT/pages/998998951/Python+3+Conversion+and+Usage+Tips), which specifies preference for using `self.assertContains` instead of explicit decoding.